docs/en/about-appium/platform-support.md

```diff
@@ -14,7 +14,7 @@ iOS automation is supported with two drivers:
 Please refer to these driver docs for setup instructions.
 
 * Versions: 9.0 and up (as a rule, Appium supports the latest two iOS versions)
-* Devices: iPhone Simulator, iPad Simulator, and real iPhones and iPads
+* Devices: iPhone Simulator, iPad Simulator and tvOS Simulator and real iPhones, iPads and tvOS
```

Comment:
multiple `and` don't looks very nice. I'd rather enumerate everything and split things with comma. Only the last item could be combined using `and`